A32NX/html_ui/Pages/A32NX_Utils/NXApi.js

```diff
@@ -14,6 +14,21 @@ class NXApi {
             });
     }
 
+    static getAirport(icao) {
+        if (!icao) {
+            throw ("No ICAO provided");
```

Comment:
You'd usually want to throw Error objects instead of literal strings:

```suggestion
            throw new Error("No ICAO provided");
```